pkg/usrlocalsharelima/usrlocalsharelima.go

```diff
@@ -27,19 +27,43 @@ import (
 // of the symlink, and not the actual executable. This is important when using Homebrew.
 //
 // If os.Args[0] is invalid, this function still falls back on os.Executable().
-var executableViaArgs0 = sync.OnceValues(func() (string, error) {
+var executableViaArgs0 = sync.OnceValues(func() (self string, _ error) {
+	self, err := os.Executable()
+	if err != nil {
+		return "", err
+	}
+
 	if os.Args[0] == "" {
 		logrus.Warn("os.Args[0] has not been set")
-	} else {
-		executable, err := exec.LookPath(os.Args[0])
-		if err == nil {
-			// LookPath() will add the `.exe` file extension on Windows, but will not return an
-			// absolute path if the argument contained any of `:/\` (or just `/` on Unix).
-			return filepath.Abs(executable)
-		}
+		return
+	}
+
+	inPath, err := exec.LookPath(os.Args[0])
```

Comment:
I'm not a fan of the name `inPath`, but it is ok if you later call the stat return value `statInPath`. But if you want to continue to call it `statFound`, then this should be

```suggestion
	found, err := exec.LookPath(os.Args[0])
```